File: Code/Prototype/Version_5/WebApplication1/ViewModels/PatientPersonalDataViewModel.cs
using BusinessLogic.Models;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace WebApplication1.ViewModels
{
    public class PatientPersonalDataViewModel
    {
        public PatientModel Info { get; set; }
        public IEnumerable<PatientAlergyModel> PatientAlergies { get; set; }
        public IEnumerable<AlergyTypeModel> AlergyTypes { get; set; }
        public PatientLocalizations PatientLocalizations { get; set; }
        public int AlergyDescriptionMaxLength { get; set; }
    }
}

File: Code/Prototype/Version_5/WebApplication1/Scripts/App/Model/PersonalData/PatientPersonalDataViewModel.js
function PatientPersonalDataViewModel(data) {
    var self = this;

    this.Info = new PatientModel(data.Info);

    this.PatientAlergies = ko.observableArray([]);

    if (data.PatientAlergies != null) {
        data.PatientAlergies.forEach(function (item) {
            self.PatientAlergies.push(new PatientAlergyModel(item));
        });
    };

    this.AlergyTypes = ko.observableArray([]);

    if (data.AlergyTypes != null) {
        data.AlergyTypes.forEach(function (item) {
            self.AlergyTypes.push(new AlergyTypeModel(item));
        });
    };

    this.PatientLocalizations = new PatientLocalizations(data.PatientLocalizations);

    this.AlergyDescriptionMaxLength = ko.observable(data.AlergyDescriptionMaxLength);

};